### PR TITLE
Obstacles spawn further away from each other.

### PR DESCRIPTION
**Pull request by LMiljak**
The distance is also randomized.

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/obstacle/ObstacleSpawner.java [hunk 1]
```diff
@@ -68,13 +68,27 @@ public ObstacleSpawner(MainEnvironment environment) {
...
+     * 		between two obstacles.
+     */
+    private float getDistanceToNextObstacle() {
+    	final Random rand = new Random();
+    	final float minimumDistance = 30.f;
+    	final float maximumDistance = 180.f;
+    	
+    	final float result = rand.nextFloat() * (maximumDistance - minimumDistance) + minimumDistance;
+    	System.out.println(result);
+    	return result;
```
Comment (dVoorhout) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/obstacle/ObstacleSpawner.java:90]: remove the logging please

**Comment by dVoorhout** [comment]
Besides that it looks much better and more playable, I approve

**Comment by LMiljak** [comment]
Whoops, forgot that log I made.
It's fixed now

**Comment by dVoorhout** [comment]
Looks better

**Comment by NilsHullegien** [comment]
Looks good, merging

**Pull request merged**